src/Omnipay/Stripe/Subscription/Message/SubscriptionRequest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Omnipay\Stripe\Subscription\Message;
+
+use Goteo\Application\Config;
+use Goteo\Library\Text;
+use Goteo\Model\Invest;
+use Goteo\Model\User;
+use Omnipay\Common\Message\AbstractRequest;
+use Stripe\Customer;
+use Stripe\Product;
+use Stripe\StripeClient;
+
+class SubscriptionRequest extends AbstractRequest
+{
+    private array $data;
+    private StripeClient $stripe;
+
+    public function __construct(array $data)
+    {
+        $this->data = $data;
+        $this->stripe = new StripeClient(Config::get('payments.stripe.secretKey'));
+    }
+
+    public function getData()
+    {
+        return $this->data;
+    }
+
+    /**
+     * @param array{user: User, invest: Invest} $data
+     */
+    public function sendData($data)
+    {
+        $price = $this->stripe->prices->create([
+            'unit_amount' => ($data['invest']->amount + $data['invest']->donate_amount) * 100,
+            'currency' => 'eur',
+            'recurring' => ['interval' => 'month'],
+            'product' => $this->getStripeProduct($data['invest'])->id
+        ]);
+
+        $session = $this->stripe->checkout->sessions->create([
+            'customer' => $this->getStripeCustomer($data['user'])->id,
+            'success_url' => $this->getRedirectUrl('/dashboard/subscriptions'),
```

Comment:
When the invest is successfull, it should be redirected to the step 3 of the invest process, where the user can update their fiscal information and so on.